scene/2d/physics/collision_shape_2d.cpp

```diff
@@ -67,13 +67,11 @@ void CollisionShape2D::_notification(int p_what) {
 				_update_in_shape_owner();
 			}
 		} break;
-
-		case NOTIFICATION_LOCAL_TRANSFORM_CHANGED: {
+		case NOTIFICATION_TRANSFORM_CHANGED: {
```

Comment:
```suggestion

		case NOTIFICATION_TRANSFORM_CHANGED: {
```
Please keep this spacing for readability